### ISSUE TITLE
Color contrast accessibility issue

### ISSUE THREAD

**Issue by eliotjordan** [pulibrary/pulmap#1085]
There is a color contrast WCAG violation on the home page with the featured content cards:

```
       1) color-contrast: Elements must have sufficient color contrast (serious)
           https://dequeuniversity.com/rules/axe/4.4/color-contrast?application=axeAPI
           The following 1 node violate this rule:

               Selector: .col-lg-6.col-12:nth-child(5) > .card > .card-body > .grid-item > p
               HTML: <p>Data representing geographic features as points, lines, and polygons.</p>
               Fix any of the following:
               - Element has insufficient color contrast of 2.99 (foreground color: #92959a, background color: #ffffff, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1
```

https://github.com/pulibrary/pulmap/blob/main/app/views/catalog/_featured_content.html.erb
